packages/babel-preset-expo/src/__tests__/jsc-bundle.test.ts

```diff
@@ -0,0 +1,57 @@
+import * as babel from '@babel/core';
+import traverse from '@babel/traverse';
+import * as t from '@babel/types';
+
+import preset from '..';
+
+it('bundles `@react-native/js-polyfills/console.js` without imports', () => {
+  const sourceFile = require.resolve('@react-native/js-polyfills/console.js');
+  const result = babel.transformFileSync(
+    sourceFile,
+    getConfig({
+      ast: true,
+      caller: getCaller({
+        name: 'metro',
+        engine: 'default',
+        platform: 'ios',
+        isDev: false,
+      }),
+    })
+  );
+
+  // Ensure the AST is available
+  expect(result?.ast).toBeDefined();
+
+  // Collect all `import` or `require` statements from the AST
+  const importRefs: Set<any> = new Set();
+  traverse(result!.ast, {
+    ImportDeclaration(path) {
+      importRefs.add(path.node);
+    },
+    CallExpression(path) {
+      if (t.isIdentifier(path.node.callee, { name: 'require' })) {
+        importRefs.add(path.node);
+      }
+    },
+  });
+
+  // Ensure there are no `import` or `require` statements
+  expect([...importRefs]).toEqual([]);
```

Comment:
Checking specifically on equality to an empty array gives us the exact imports when failing:

![image](https://github.com/user-attachments/assets/dcd5dae6-974a-4c09-840c-0788ae0f716a)